servers/rendering/renderer_rd/storage_rd/render_scene_buffers_rd.cpp

```diff
@@ -733,7 +733,11 @@ uint32_t RenderSceneBuffersRD::get_color_usage_bits(bool p_resolve, bool p_msaa,
 }
 
 RD::DataFormat RenderSceneBuffersRD::get_depth_format(bool p_resolve, bool p_msaa, bool p_storage) {
-	if (p_resolve) {
+	// TODO Our rendering engine does not support just having a depth attachment, it always assumed we combine with stencil.
```

Comment:
This does need to be solved. I ran into a snag that our driver system assumes that any depth texture MUST BE a depth + stencil texture, and as our resolve texture is just a depth texture, it wouldn't accept it.

This workaround for now is fine, but ideally we should add support for either:
- `TEXTURE_USAGE_DEPTH_ATTACHMENT_BIT` so that we can setup pipelines with just normal depth textures (could be useful for local rendering devices and such)
- `TEXTURE_USAGE_DEPTH_RESOLVE_ATTACHMENT_BIT` so we can properly recognise our texture as a resolve texture.

Reply:
I ended up implementing `TEXTURE_USAGE_DEPTH_RESOLVE_ATTACHMENT_BIT`